examples/tcp_server.rs

```diff
@@ -112,33 +112,42 @@ fn handle_connection_event(
     }
 
     if event.is_readable() {
+        let mut connection_closed = false;
+        let mut received_data = Vec::with_capacity(4056);
```

Comment:
Can you change this to 4096 (4056 is an odd capacity).

Reply:
:man_facepalming: